academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -2,27 +2,1410 @@
 title: Create index from Microsoft Academic Graph(MAG)
 description: Step-by-step guide for generating MAKES indexes from a MAG release.
 ms.topic: tutorial
-ms.date: 2020-02-10
+ms.date: 2020-02-19
 ---
 
 # Generating MAKES indexes from a MAG release
 
- Step-by-step guide for generating MAKES indexes from a Microsoft Academic Graph(MAG) release. For more information on obtaining a MAG release, visit [Get Microsoft Academic Graph on Azure storage](../graph/get-started-setup-provisioning.md)
+ Step-by-step guide for generating MAKES indexes from a Microsoft Academic Graph(MAG) release. For more information on obtaining a MAG release, visit [Get Microsoft Academic Graph on Azure storage](../graph/get-started-setup-provisioning.md).  In this example we will be building a custom index of all Microsoft publications and associated entities.
 
 ## Prerequisites
 
-- Microsoft Academic Graph(MAG) release
-- Microsoft Academic Knowledge Exploration Service(MAKES) release
-- Azure subscription
+- [Microsoft Academic Graph(MAG) subscription](../graph/get-started-setup-provisioning.md)
+- [Azure Data Lake Analytics (ADLA) setup with MAG](../graph/get-started-setup-azure-data-lake-analytics.md)
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
 
 ## Generating MAKES entities using USQL
 
+MAKES indexes are generated from specifically formatted MAG data.  In order to create a new index, you will need to generate the entities you would like in the index from MAG.  We will start by running a USQL script on your MAG subscription to generate the data required to build the custom index.  To do this, you will need to go to your Azure Data Lake Analytics (ADLA) service instance and submit a new ALDA job to generate the text files containing academic data.
```

Comment:
>MAKES indexes are generated from specifically formatted MAG data.  In order to create a new index, you will need to generate the entities you would like in the index from MAG.  We will start by running a USQL script on your MAG subscription to generate the data required to build the custom index. [](start = 0, length = 297)

MAKES requires data it indexes to be placed in a single JSON file, with each line representing an individual entity. For this example, we will use a U-SQL script to filter and transform MAG data into JSON data which will then be used to create a custom index.